data/tokens.json

```diff
@@ -2326,13 +2326,30 @@
       "value": "{Text.gtmds_color_text-color-subdued}",
       "type": "color"
     },
+    "gtmds_dropdown-group-header_color": {
```

Comment:
yes one for the typography and one for the color scoped for the dropdown as I didn't see such tokens for the action item did I duplicate something unintentionally?

Reply:
No, I was wondering if should Action-item-title should adopt "dropdown" as a name or the opposite?

Reply:
Honestly I don't know if an action item component will exist or it will be implicitly defined in other areas and that whole thing. We would probably need to discuss that with the engineers.